src/sections/Partners/Partners.module.scss

```diff
@@ -0,0 +1,68 @@
+@import "../../styles/vars.scss";
+
+.section {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  max-width: 640px;
+
+  a {
+    text-decoration: none;
+  }
+}
+
+.partnersIconsWrapper {
```

Comment:
Maybe we could give it a little bit more space at the bottom? To make the spacing equal to the spacing we have above the title?

![Screenshot 2021-12-15 at 18 39 49](https://user-images.githubusercontent.com/20808724/146237882-7c26b261-6fdd-4d84-ab3b-49efed8ec44c.png)